### PR TITLE
Documentation issue read me.

### PR DESCRIPTION
**Pull request by SANJAY-NT**
## Related Issue 

- The download image under official Build in readme is not displayed

- [x] Wanted  display correction

### Describe the changes I've made

The issue was there should be a image displayed next to Android download in official builds  under Continuous build status, but so.
This may have been caused because the image or gif link may have expired or changed. 

## Type of change
I went through the code and found the link was showing 403 error and it may be the cause of the image not being displayed.

What sort of change have you made:

- [x] Code style update (formatting, local variables)
- [x] This change requires a documentation update

## How Has This Been Tested?
The change I made is only related to the display change and none of the other lines were changed in any manner and hence the functionalities remain the same.

Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have not made corresponding changes to the documentation.

 ## Detailed explanation of the code lines changed
-The previous lines before change line no. 118
``` **Android**                   | [Status](https://storage.googleapis.com/tensorflow-kokoro-build-badges/android.html)                                 | [Download](https://bintray.com/google/tensorflow/tensorflow/_latestVersion)```

previous

-The change to the code lines 
``` **Android**                   | [Status](https://storage.googleapis.com/tensorflow-kokoro-build-badges/android.html)                                 | [Download](https://bintray.com/google/tensorflow/tensorflow/_latestVersion) ```

after

- The above is the relative change i would like to suggest to update the documentation.
### Sanjay NT

### CODE REVIEW

**Comment by gbaned** [comment]
@SANJAY-NT  Can you please sign CLA. Thanks!

File: README.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ Build Type                    | Status
...
 **Windows GPU**               | [![Status](https://storage.googleapis.com/tensorflow-kokoro-build-badges/windows-gpu.svg)](https://storage.googleapis.com/tensorflow-kokoro-build-badges/windows-gpu.html)                         | [PyPI](https://pypi.org/project/tf-nightly-gpu/)
-**Android**                   | [![Status](https://storage.googleapis.com/tensorflow-kokoro-build-badges/android.svg)](https://storage.googleapis.com/tensorflow-kokoro-build-badges/android.html)                                 | [![Download](https://api.bintray.com/packages/google/tensorflow/tensorflow/images/download.svg)](https://bintray.com/google/tensorflow/tensorflow/_latestVersion)
+**Android**                   | [![Status](https://storage.googleapis.com/tensorflow-kokoro-build-badges/android.svg)](https://storage.googleapis.com/tensorflow-kokoro-build-badges/android.html)                                 | [Download](https://bintray.com/google/tensorflow/tensorflow/_latestVersion)
```
Comment (MarkDaoust) [inline README.md:118]: > https://bintray.com/google/tensorflow/tensorflow/_latestVersion

That link just redirects to this web page:

https://jfrog.com/center-sunset/

Is this maintained? Maybe it's time to delete?

File: README.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ Build Type                    | Status
...
-**Android**                   | [![Status](https://storage.googleapis.com/tensorflow-kokoro-build-badges/android.svg)](https://storage.googleapis.com/tensorflow-kokoro-build-badges/android.html)                                 | [![Download](https://api.bintray.com/packages/google/tensorflow/tensorflow/images/download.svg)](https://bintray.com/google/tensorflow/tensorflow/_latestVersion)
+**Android**                   | [![Status](https://storage.googleapis.com/tensorflow-kokoro-build-badges/android.svg)](https://storage.googleapis.com/tensorflow-kokoro-build-badges/android.html)                                 | [Download](https://bintray.com/google/tensorflow/tensorflow/_latestVersion) 
 **Raspberry Pi 0 and 1**      | [![Status](https://storage.googleapis.com/tensorflow-kokoro-build-badges/rpi01-py3.svg)](https://storage.googleapis.com/tensorflow-kokoro-build-badges/rpi01-py3.html)                             | [Py3](https://storage.googleapis.com/tensorflow-nightly/tensorflow-1.10.0-cp34-none-linux_armv6l.whl)
```
Comment (MarkDaoust) [inline README.md:119]: I have a similar question for these Raspberry Pi links. "tensorflow-1.10" "cp34" this looks really out of date.

**Review by MarkDaoust — CHANGES_REQUESTED** [review]
THis probably needs some additional cleanup.

File: README.md (hunk 2 above)
Comment (SANJAY-NT) [reply README.md:119]: @MarkDaoust  Yes it looks so. Need a review and update.

File: README.md (hunk 1 above)
Comment (SANJAY-NT) [reply README.md:118]: @MarkDaoust  @gbaned  Thank you for your reply. This is my first time here and I don't know the rules and regulations. I would like to work on the issue and go through the process. Can I get some guidance from you how to proceed further.

**Comment by SANJAY-NT** [comment]
@googlebot I signed it!

File: README.md (hunk 2 above)
Comment (SANJAY-NT) [reply README.md:119]: @MarkDaoust where can I find the valid links to update the read.me.

**Comment by gbaned** [comment]
@SANJAY-NT  Can you please resolve conflicts? Thanks!

**Comment by SANJAY-NT** [comment]
The issue is resolved So i would like to close this pull request.

**Comment by SANJAY-NT** [comment]
@gbaned  I resolved the conflict.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
This now duplicates the existing table

**Comment by SANJAY-NT** [comment]
> This now duplicates the existing table

Yes, by the time I completed the fixed it was already updated by someone else so the issue is over.

**Pull request closed without merging**